src/chapters/git/remote-repository.rst

```diff
@@ -0,0 +1,182 @@
+Remote Repositories
+===================
+
+In this chapter, we learned how to use the Git version control system to keep
+backups of our code. This version history (the repository) is stored on our
```

Comment:
Thinking about this a bit, I prefer "snapshots" to "backups," given the connotation of the latter term as being primarily to keep safe version of a file or project in case we lose the primary version. VCS do this, but it's only a portion of what they're for. Just an opinion, though. Take it or leave it.

Reply:
I went with "snapshots".